src/Stratis.Bitcoin.Features.Wallet/Controllers/WalletController.cs

```diff
@@ -784,6 +784,45 @@ public IActionResult GetUnusedAddresses([FromQuery]GetUnusedAddressesModel reque
             }
         }
 
+        /// <summary>
+        /// Gets the specified number of unused addresses.
+        /// </summary>
+        [Route("addresses")]
+        [HttpGet]
+        public IActionResult GetAllAddresses([FromQuery]GetAllAddressesModel request)
```

Comment:
Why do we need such a method I wonder?

Reply:
because at the moment, users have no idea what addresses they have in their wallet.
All they can ever know is the first unused address.
This gives them a bit more data and POWER.

Reply:
ok cool so push when tests are done.